journal/modernized/08-chapter-8.adoc

```diff
@@ -273,35 +273,35 @@ for to Him alone the praise is due.
 It may be a preparatory baptism for the ensuing Quarterly Meeting, +++[+++and if so]
 I am satisfied to abide under its continuance, desiring that the Lord will not pity,
 nor His hand spare,
-till His holy hand has formed and fashioned me to what lie would have me to be.
+till His holy hand has formed and fashioned me to what He would have me to be.
 My Lord and Master cried out on the cross, "`Why have you forsaken me?`"
 I have in similar and lesser circumstances, cried out in like manner,
 but did not feel any condemnation, therefore I am encouraged to hold on my way,
 hoping to be enabled to do so, unto the end of the race,
 which is not gained by the swift, but those who hold out to the end will gain the prize.
 Herein the creaturely part is crucified, and the Lord magnified,
 because His mercy has endured so long.
-I believe I may say with the apostle, "`lam crucified with Christ, nevertheless I live,
+I believe I may say with the apostle, "`l am crucified with Christ, nevertheless I live,
```

Comment:
This should be changed to `I am` -- it's currently `l am` (with a lowercase L) -- which in code font looks super similar.